app/src/activity.c

```diff
@@ -126,7 +130,14 @@ void activity_work_handler(struct k_work *work) {
     int32_t current = k_uptime_get();
     int32_t inactive_time = current - activity_last_uptime;
 #if IS_ENABLED(CONFIG_ZMK_SLEEP)
-    if (inactive_time > MAX_SLEEP_MS && !is_usb_power_present()) {
+    if (inactive_time > MAX_SLEEP_MS &&
+        !is_usb_power_present()
```

Comment:
I hate this formatting with the first `&&` on the same line but the second one (`#if`) being on the next one...

Not consistent, but it is what clangd-format wanted